package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=16"
   },
-  "version": "7.70.29",
+  "version": "7.70.30",
```

Comment:
Based on when this is ready for merge, we will be changing this tag to the latest unreleased one.